Sprint-2/implement/cases.js

```diff
@@ -13,3 +13,9 @@
 
 // You will need to come up with an appropriate name for the function
 // Use the string documentation to help you find a solution
+function toUpperCase(string) {
```

Comment:
This function name makes sense, but you're also adding underscores `_` to your string, so it's doing slightly more than just converting the string to uppercase!

The commented instructions at the top give a clue to the naming typically used for this name used for underscores between the words in all upper case

If you're interested in other types of cases and their names, this is a helpful article: https://verbalraj.medium.com/case-types-in-programming-918770f82440